staging/src/k8s.io/api/policy/v1beta1/types.go

```diff
@@ -84,6 +84,8 @@ type PodDisruptionBudgetStatus struct {
 // +k8s:prerelease-lifecycle-gen:introduced=1.5
 // +k8s:prerelease-lifecycle-gen:deprecated=1.22
 
+// DEPRECATED - This group version of PodDisruptionBudget is deprecated by policy/v1/PodDisruptionBudget. See
+// the release notes for more information.
```

Comment:
Not needed, this is handled by the above tags automatically.

Reply:
Removed